my-site.rb

```diff
@@ -3,6 +3,30 @@
 class MySite < Sinatra::Base
 
   get "/" do
-
+    erb :index
+  end
```

Comment:
Add white space after each `end` to denote the separation between routes so after lines 10, 13, 16, etc.